src/OpenTelemetry/.publicApi/net452/PublicAPI.Shipped.txt

```diff
@@ -55,7 +55,7 @@ OpenTelemetry.Resources.ResourceBuilder
 OpenTelemetry.Resources.ResourceBuilder.Build() -> OpenTelemetry.Resources.Resource
 OpenTelemetry.Resources.ResourceBuilder.Clear() -> OpenTelemetry.Resources.ResourceBuilder
 OpenTelemetry.Resources.ResourceBuilderExtensions
-OpenTelemetry.Sdk
+OpenTelemetry.OpenTelemetrySdk
```

Comment:
We cannot break public API, as the library has released stable 1.0 version and breaking changes can only be added with a major version bump.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/VERSIONING.md